crates/astria-cli/src/commands/bridge/collect.rs

```diff
@@ -199,10 +199,21 @@ async fn block_to_actions(
         .await
         .wrap_err_with(|| {
             format!(
-                "failed getting actions for block; block hash: `{block_hash}`, block height: \
+                "failed getting logs for block; block hash: `{block_hash}`, block height: \
```

Comment:
This is not correct. The `actions_fetcher` is promising to give you actions. Why is this talking about logs? That's confusing.